### PR TITLE
Autoplot - Add messagebox for emptytreewidget

### PR DESCRIPTION
**Pull request by Preetam-Das26**
**Purpose of PR?**:
Autoplot - Add messagebox for emptytreewidget
Fixes #

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Does this PR results in some Documentation changes?**
_If yes, include the list of Documentation changes_

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [ ] New feature (Non-API breaking changes that adds functionality)
- [ ] PR Title follows the convention of  `<type>: <subject>`
- [ ] Commit has unit tests

### CODE REVIEW

File: mslib/msui/autoplot_dockwidget.py [hunk 1]
```diff
@@ -298,7 +298,12 @@ def add_to_treewidget(self, parent, parent2, config_settings, treewidget, flight
             config_settings["automated_plotting_flights"].append([flight, sections, vertical, filename, itime, vtime])
             parent.refresh_signal_emit.emit()
         if treewidget.objectName() == "autoplotSecsTreeWidget":
-            if url is None:
+            if url == "":
+                QMessageBox.information(
+                    self,
+                    "WARNING",
+                    "Please select the URL,layer,styles and level(row information first)"
```
Comment (ReimarBauer) [inline mslib/msui/autoplot_dockwidget.py]: the text linter has not seen that, but we should do it :)

Please select the URL, layer, styles and level (row information first)"

File: mslib/msui/autoplot_dockwidget.py [hunk 2]
```diff
@@ -346,7 +351,12 @@ def update_treewidget(self, parent, parent2, config_settings, treewidget, flight
             parent.refresh_signal_emit.emit()
 
         if treewidget.objectName() == "autoplotSecsTreeWidget":
-            if url is None:
+            if url == "":
+                QMessageBox.information(
+                    self,
+                    "WARNING",
+                    "Please select the URL,layer,styles and level(row information first)"
```
Comment (ReimarBauer) [inline mslib/msui/autoplot_dockwidget.py]: Please select the URL, layer, styles and level (row information first)"

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
small text style changes wanted :)

File: mslib/msui/autoplot_dockwidget.py (hunk 1 above)
Comment (Preetam-Das26) [reply mslib/msui/autoplot_dockwidget.py]: Done

File: mslib/msui/autoplot_dockwidget.py (hunk 2 above)
Comment (Preetam-Das26) [reply mslib/msui/autoplot_dockwidget.py]: done

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**